db/migrate/20210325230116_add_expanded_covid_registration.rb

```diff
@@ -0,0 +1,26 @@
+class AddExpandedCovidRegistration < ActiveRecord::Migration[6.0]
+  def change
+    create_table :covid_vaccine_expanded_registration_submissions, id: :serial do |t|
+      t.string "submission_uuid", null: false
+      t.string "vetext_sid"
+      t.boolean "sequestered", default: true, null: false
+      t.string "state"
+      t.string "email_confirmation_id"
+      t.string "enrollment_id"
```

Comment:
enrolled_at might be useful for audit purposes.

Reply:
Since the enrollment is asynchronous (via flat files), I don't know that a timestamp will be that meaningful. I will make sure the eligibility_info includes a timestamp of when the eligibility info was fetched from MPI/EMIS.